components/style/gecko_bindings/check_bindings.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+LICENSE = """/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Yes, `\` followed by a newline is an escape sequence that [is ignored/skipped by Python](https://docs.python.org/2/reference/lexical_analysis.html#string-literals), so the resulting string won't have a leading newline.

Try it out in a small test file!